apps/synapse-portal-framework/src/components/csbc-home-page/Ecosystem.tsx

```diff
@@ -1,5 +1,5 @@
 import { useState } from 'react'
-import { SynapseComponents } from 'synapse-react-client'
```

Comment:
Try to replace these `SynapseComponents` exports -- these are really bad -- they will trigger bundling/loading almost every component in SRC